### ISSUE TITLE
Cheat codes are not the original

### ISSUE THREAD

**Issue by tmr83** [the3dfxdude/7kaa#259]
I have the cheat codes in the manual, and they are different when compared to cheat codes documented on the old Realms of the Kingdoms site. We talked about this a bit on the forums one time. I'm fuzzy on the details, but I do remember you saying back then that you prefer the original codes.

For example,

Current manual: n - Add all technology.

Realms of the Kingdoms: T - All technological advances and scrolls.

Current manual: z - Construct building instantly.

Realms of the Kingdoms: B - Finish construction of selected building immediately.

**Comment by the3dfxdude** [comment]
Hotkeys were adjusted to support additional hotkeys. My wish was to make sure all the original manual and keycard keys stayed the same. Most did I think.

Cheat codes come in on the next, lower level priority. I am not sure if we should or shouldn't. If they are cheat codes that were shared by the original developers, I would say we should. I wouldn't rely on the current manual as I think that is just what we have right now. If you want to address this, find what cheat key was changed to support what hotkey action. And list that out. Then identify a strategy to use to support cheats or hotkeys.

In the past I suggested to replace the cheat hotkeys with a cheat menu. I still need to clean up the config and gui code, which is approximately the next up (well and some level of save file adjustment to finally get the job done).
